packages/wonder-blocks-switch/src/themes/default.ts

```diff
@@ -0,0 +1,66 @@
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+
+const theme = {
+    color: {
+        bg: {
+            primary: {
+                off: tokens.color.offBlack50,
+                disabledOff: tokens.color.offBlack32,
+                activeOff: tokens.color.offBlack64,
+                on: tokens.color.blue,
+                disabledOn: tokens.color.fadedBlue,
+                activeOn: tokens.color.activeBlue,
+            },
+            secondary: {
+                on: tokens.color.white,
+                off: tokens.color.white,
+            },
+            tertiary: {
+                on: tokens.color.blue,
+                disabledOn: tokens.color.fadedBlue,
+                off: tokens.color.offBlack50,
+                disabledOff: tokens.color.offBlack32,
+            },
+        },
+        outline: {
+            default: tokens.color.blue,
+        },
+    },
+    border: {
+        radius: {
+            // slider
+            small: tokens.spacing.small_12,
+            // switch
+            full: tokens.border.radius.full,
+        },
+    },
+    size: {
+        height: {
+            none: 0,
+            // switch
+            medium: 20,
+            // slider
+            large: tokens.spacing.large_24,
+        },
+        width: {
+            none: 0,
+            xxxxSmall: tokens.spacing.xxxxSmall_2,
+            // NOTE: This token is specific to the Switch component
+            // switch
+            medium: 20,
+            // NOTE: This token is specific to the Switch component
+            // slider
+            xLarge: 40,
```

Comment:
**suggestion/nit:** Following up on the `width` names...

In this case, these component tokens could be defined in a scalar way: `small`, `medium`, `large`, so there are no gaps in the t-shirt size and feels more linear. Wdyt?

```suggestion
            small: tokens.spacing.xxxxSmall_2,
            // NOTE: This token is specific to the Switch component
            // switch
            medium: 20,
            // NOTE: This token is specific to the Switch component
            // slider
            large: 40,
```

Reply:
that makes sense to me.